sdk/appconfiguration/app-configuration/src/internal/synctokenpolicy.ts

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { RequestPolicy, RequestPolicyOptions, BaseRequestPolicy, WebResource, HttpOperationResponse, RequestPolicyFactory } from "@azure/core-http";
+
+/**
+ * The sync token header, as described here:
+ * https://github.com/Azure/AppConfiguration/blob/master/docs/REST/consistency.md
+ */
+export const SyncTokenHeaderName = "Sync-Token";
```

Comment:
You may need to add @internal or @ignore to the new symbols you're exporting just so TypeDoc doesn't find and include them in the docs.

Reply:
So none of these are actually intended to be set or interacted with by external clients.

One issue I had is I did want to make SyncTokens injectable, but only for internal testing. So I added in an AppConfigurationClientOptions interface which has a syncTokens field. I typed the field as 'any' to avoid having to export SyncTokens.